tools/iana_to_rflx.py

```diff
@@ -0,0 +1,214 @@
+            name = f"{registry_title}_{name}"
+        DUPLICATES.append(name)
+
+        if record.comment:
+            for comment_line in record.comment:
+                file.write(f"{'':<7}-- {comment_line}\n")
+            file.write(f"{'':<7}--\n")
+        file.write(f"{'':<7}{f'{name} => {record.value}'},\n\n")
+
+    file.seek(file.tell() - 3)
+    file.write(")\n")
+    file.write(f"{'':<3}with Size => {size}")
+    if always_valid and len(normalized_records).bit_length() == size:
+        file.write(", Always_Valid;")
+    else:
+        file.write(";")
+    file.write("\n\n")
+
+
+def _normalize_records(records: List[Element], registry_name: str) -> List["Record"]:
+    normalized_records: Dict[str, Record] = {}
+    for record in records:
+        name_tag = _get_name_tag(record)
+        value_tag = "value"
+
+        name_element = record.find(f"iana:{name_tag}", NAMESPACE)
+        value_element = record.find(f"iana:{value_tag}", NAMESPACE)
+
+        if name_element is None or value_element is None:
+            continue
+
+        assert isinstance(name_element.text, str)
+        assert isinstance(value_element.text, str)
+        name = name_element.text
+        value = value_element.text
+
+        if re.search(r"RESERVED|UNASSIGNED", name, flags=re.I):
+            continue
+
+        comment = [
+            element
+            for element in record.iterfind("*", NAMESPACE)
+            if element.tag
+            not in [f"{{{NAMESPACE['iana']}}}{name_tag}", f"{{{NAMESPACE['iana']}}}{value_tag}"]
+        ]
+
+        r = Record(name, value, comment)
+        if value in normalized_records:
+            normalized_records[value].join(r, registry_name)
+        else:
+            normalized_records[value] = r
+    return list(normalized_records.values())
+
+
+def _get_name_tag(record: Element) -> str:
+    sub_elements = record.findall("*", NAMESPACE)
+    child_names = set(c.tag[c.tag.index("}") + 1 :] for c in sub_elements)
+    possible_name_tags = ["name", "code", "type", "description"]
+    for tag in possible_name_tags:
+        if tag in child_names:
+            return tag
+    return ""
+
+
+class Record:
+    def __init__(self, name: str = "", value: str = "", comments: Optional[List[Element]] = None):
+        self.name = _normalize_name(name)
+        self.value, self.bit_length = _normalize_value(value)
+        self.comment_list = comments or []
+        self.alternative_names: List[str] = []
+
+    def join(self, duplicate: "Record", registry_name: str) -> None:
+        self.name = f"{registry_name}_{self.value.replace('#', '_')}"
+        self.comment_list.extend(duplicate.comment_list)
+        self.alternative_names.append(f"alternative_name = {duplicate.name}")
+
+    @property
+    def comment(self) -> List[str]:
+        return _normalize_comment(self.comment_list, self.alternative_names)
+
+
+def _normalize_comment(comment_list: List[Element], alternative_names: List[str]) -> List[str]:
```

Comment:
Could be a method of the `Record` class. And could be joined with the `comment` method.